Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -55,7 +55,13 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $connection = $this->getDoctrineConnection($input->getOption('connection'));
+        if (empty($input->getOption('connection'))) {
+            $connectionName = $this->getContainer()->get('doctrine')->getDefaultManagerName();
```

Comment:
Well, there is an issue currently indeed. But your fix is wrong. Connection names are manager names are not guaranteed to match (even if the shortcut notation uses ``default`` as name for both of them).

The issue is that currently, the message says `` Created database for connection named `symfony_test2` `` but the name it displays is the DB name, not the connection name